lib/models/asset-size-printer.js

```diff
@@ -9,56 +9,60 @@ function AssetPrinterSize(options) {
-      var gzip = Promise.denodeify(zlib.gzip);
-      var files = this.makeFileGlob();
-      var testFileRegex = /(test-(loader|support))|(testem)/i;
-      var promises = [];
-
-      this.validateAssetPath(files);
-      ui.writeLine(chalk.green('File sizes:'));
-
-      files
-        // Skip test files
-        .filter(function (file) {
-          var filename = path.basename(file);
-          return !testFileRegex.test(filename);
-        })
-        // Print human-readable file sizes (including gzipped)
-        .forEach(function (file) {
-          var filename = path.basename(file);
-          var contentsBuffer = fs.readFileSync(file);
-          var gzipPromise = gzip(contentsBuffer).then(function (buffer) {
-            return {
-              name: filename,
-              size: filesize(contentsBuffer.length),
-              gzipSize: filesize(buffer.length),
-              showGzipped: contentsBuffer.length > 0
-            };
-          });
-
-          promises.push(gzipPromise);
-        });
-
-      return Promise.all(promises);
-    }.bind(this))
-    .then(function (files) {
-      files.forEach(function (file) {
-        var sizeOutput = file.size;
-        if (file.showGzipped) {
-          sizeOutput += ' (' + file.gzipSize + ' gzipped)';
-        }
-
-        ui.writeLine(chalk.blue(' - ' + file.name + ': ') + chalk.white(sizeOutput));
+AssetPrinterSize.prototype.print = function () {
+  return this.makeAssetSizesObject().then(function (files) {
+
+    var ui = this.ui;
+
+    ui.writeLine(chalk.green('File sizes:'));
+    return files.forEach(function (file) {
+      var sizeOutput = file.size;
+      if (file.showGzipped) {
+        sizeOutput += ' (' + file.gzipSize + ' gzipped)';
+      }
+
+      ui.writeLine(chalk.blue(' - ' + file.name + ': ') + chalk.white(sizeOutput));
+    });
+
+  }.bind(this));
+};
+
+AssetPrinterSize.prototype.makeAssetSizesObject = function () {
+  var filesize = require('filesize');
+  var fs = require('fs');
+  var zlib = require('zlib');
+  var gzip = Promise.denodeify(zlib.gzip);
+  var files = this.makeFileGlob();
+  var testFileRegex = /(test-(loader|support))|(testem)/i;
+  var promises = [];
+
+  this.validateAssetPath(files);
+
+  files
+    // Skip test files
+    .filter(function (file) {
+      var filename = path.basename(file);
+      return !testFileRegex.test(filename);
+    })
+    // Print human-readable file sizes (including gzipped)
+    .forEach(function (file) {
+      var filename = path.basename(file);
+      var contentsBuffer = fs.readFileSync(file);
+      var gzipPromise = gzip(contentsBuffer).then(function (buffer) {
+        return {
+          name: filename,
+          size: filesize(contentsBuffer.length),
+          gzipSize: filesize(buffer.length),
+          showGzipped: contentsBuffer.length > 0
+        };
       });
+
+      promises.push(gzipPromise);
     });
+
+  return Promise.all(promises).then(function (assetObject) {
+    return assetObject;
```

Comment:
assetObject would actually be the fullfillment of all the promises, is this variable named incorrectly or.. ?